Formula/m/micromamba.rb

```diff
@@ -8,7 +8,9 @@ class Micromamba < Formula
 
   livecheck do
     url :stable
-    regex(/^micromamba[._-]v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest do |json|
+      json["name"]
```

Comment:
```suggestion
    strategy :github_latest do |json, regex|
      match = json["name"]&.match(regex)
      next if match.blank?

      match[1]
```

Using `json["name"]` alone assumes that upstream will always use a release title that only includes the version text (e.g., `1.2.3`). We can keep this from breaking if/when they ever include any other text in the release title by matching within it using the default strategy regex instead.